deb/common/docker-ce.docker.service

```diff
@@ -0,0 +1 @@
+engine/contrib/init/systemd/docker.service
```

Comment:
This symlink ends up in `debian/` so needs to be `../` prefixed, right? :eyes:

```suggestion
../engine/contrib/init/systemd/docker.service
```

Reply:
Curious that the CI builds succeeded; I bet they're missing systemd service files :grimacing:

Reply:
oh, interesting; so this was a straight revert of your original one; wondering if things were broken there, or is it because we moved things?

Reply:
Oh, it was probably broken there!! :facepalm:

Reply:
I haven't checked yet 🙈 

I'm always confused with these paths as well, because it's possible we put the directories in a different path during build (which I need to check)

Reply:
Yeah, all these files end up under `debian/`, of which `engine/` is a "sibling directory" :sweat_smile: